cfg/rippled-example.cfg

```diff
@@ -1173,7 +1173,7 @@ filter_bits=12
 cache_mb=256
 file_size_mb=8
 file_size_mult=2
-online_delete=2000
+online_delete=30000
```

Comment:
I can take or leave this. I'm curious what the rationale for this value is.

Reply:
I picked that number based on the Capacity Planning Guide [here](https://xrpl.org/capacity-planning.html#recommendation-1) so that more than 1 day of ledger versions is stored.